### PR TITLE
QEMU: cleanup config

### PR DESCRIPTION
**Pull request by jforissier**

### CODE REVIEW

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: kconfigs/qemu.conf [hunk 1]
```diff
@@ -1,18 +1,9 @@
 CONFIG_TEE=y
 CONFIG_OPTEE=y
 CONFIG_STAGING=y
```
Comment (etienne-lms) [inline kconfigs/qemu.conf:3]: i think you can remove `CONFIG_STAGING=y`

Comment (jforissier) [reply kconfigs/qemu.conf:3]: Yes, thanks

**Comment by etienne-lms** [comment]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`

**Pull request merged**